tests/test_app/templates/helper_test.twig

```diff
@@ -0,0 +1,2 @@
+{{ helper_TestSecond_useElement() }}
+{{ helper_Text_autoParagraph('I love CakePHP') }}
```

Comment:
Some other options that may be worse or better could be:

* `helper:Text:autoParagraph(...)` Is visually smaller but still the same amount of letters.
* `helper_text_autoParagraph(...)` Using camelBack for the variables instead of PascalCase.
* `h:Text:autoParagraph(...)` Shorter prefix?

Reply:
Supporting `:` char would required adding addition token parsing code. Currently replacing the underscores with colons generates error `Twig\Error\SyntaxError: Unexpected token "punctuation" of value ":" ("end of print statement" expected).` Not really looking forward to adding more customizations if not really necessary.

> helper_text_autoParagraph(...) Using camelBack for the variables instead of PascalCase.

Using `text` instead of `Text` would mean unnecessary additional inflection to find matching helper property.

Reply:
We could go with `h_Text_autoParagraph()` to reduce typing.

Reply:
I think making the helper name and function look similar to other cake documentation/code is best so `Text_autoParagraph` looks similar to `Text->autoParagraph()`.

Reply:
Can you create Twig autocomplete on such functions? Then it would be similar again to normal PHP templating.
But I don't know enough about the IDE compatibility here.

Reply:
https://www.jetbrains.com/help/phpstorm/symfony-twig.html

I doubt any IDE would provide autocompletion for custom functions.

Reply:
I see, someone would need to port https://plugins.jetbrains.com/plugin/7219-symfony-support to cakephp version then.